boards/st/stm32f469i_disco/Kconfig.defconfig

```diff
@@ -9,5 +9,24 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
+if DISPLAY
+
+# MEMC needs to be enabled in order to store
+# display buffer to external SDRAM connected to FMC
+config MEMC
+	default y
+
+# Initialize after LTDC and MIPI-DSI
+config DISPLAY_NT35510_INIT_PRIORITY
+	default 87
```

Comment:
This should be in `boards/shields/st_b_lcd40_dsi1_mb1166/boards/st_b_lcd40_dsi1_mb1166_a09/stm32f469i_disco.conf`
& should be replaced by
```
CONFIG_DISPLAY_NT35510_INIT_PRIORITY=87
```